microsoft/electionguard-python
# #697 Export Election Record

### Issue

Fixes #697 

### Description
Adds the ability to export an election record to the GUI after a decryption has been performed.

### Testing
1. Create an election
2. Perform a decryption

Once the decryption has been completed the following button will now show up on the decryption page:

![image](https://user-images.githubusercontent.com/1769905/181653634-07e6c220-2288-4a6f-a3e9-ec6acc100698.png)

3. Click the button, you should end up on the new export election record page:

![image](https://user-images.githubusercontent.com/1769905/181653731-f8115b3d-fcb4-4abc-9381-7b73c5b5f5c3.png)

4. Click the export button

Expected: you should get an encryption package at the expected location and the screen should look like this:

![image](https://user-images.githubusercontent.com/1769905/181653816-a86012e9-9ed7-4f53-a0eb-4856181299dd.png)
